AK/HashTable.h

```diff
@@ -31,6 +31,12 @@
 #include <AK/Types.h>
 #include <AK/kmalloc.h>
 
+#ifdef KERNEL
+#    include <Kernel/StdLib.h>
+#else
+#    include <string.h>
+#endif
```

Comment:
The point of using `__builtin_memset` in AK headers is precisely to avoid the `#ifdef KERNEL` stuff. The compiled output is exactly the same, `__builtin_memset` becomes a call to `memset` anyway.

Reply:
shall I drop these commits then?

Reply:
Yes please :)